docs/source/index.rst

```diff
@@ -165,6 +165,7 @@ Table of contents
    guides/performance
    guides/parallel
    guides/deployment
+   guides/specifying_primitive_options
```

Comment:
let's actually add this under tuning_dfs. i think it's a better to be a bit higher in this list